added threatsites 21 - 24 at range 2 (tunguska, strela, manpad)

steven-omaha opened the pull request:
This addresses issue #45. I have added 4 new threatsites to range 2, numbered 21 - 24. I could not use TS 14 - 17, since some of those are already in use on other ranges. I decided to skip a few numbers and start with 21.

The 617th requires somthing like this for TAC-04.

A quick test on a local multiplayer server indicates this should work. The SAMs activate if, and only if, they are activated in the F10-menu and I set them to alert state red. I didn't put an aircraft around them for a more extensive test.


From top to bottom: 
- TS 21: Tunguska
- TS 22: Strela 1
- TS 23: Strela 10
- TS 24: MANPADs
![range 2 updated](https://cdn.discordapp.com/attachments/605831249587732497/672544255478071321/range2.png)

132nd-Entropy commented:
2 issues with this. 

1. the way you write into the script needs to be changed, you cannot have menus with more than 6-7 max entries in a given menu, or the menus can disappear for some people (known bug) 

2. when I merge your PR, the sites get renamed to 'dictkey...xxx' so you get scripting errors.

steven-omaha commented:
Copy on 1, will adapt it.

What about 2? What could cause this? All I did was copy the SA-15 into the 4 groups, renamed them, and changed the units.

132nd-Entropy commented:
I dont know what causes it, we usually dont use the PR feature. Best you download the current build, fix the names that got changed, and then use EMFT to make a new build, or if unfamiliar with the process, send the new miz to me.

steven-omaha commented:
Link to EMFT, please?

132nd-Entropy commented:
https://bwstaff-my.sharepoint.com/:u:/g/personal/rainer_beck_bwstaff_de/EZ-6cf2F6LpCvwv1pPkuwMIB1n22qJ75pYnzIfAWRbjkVw?e=gpScVc
